common/inc/sgx_tseal.h

```diff
@@ -47,18 +47,18 @@
 
 typedef struct _aes_gcm_data_t
 {
-    uint32_t  payload_size;                   /*  0: Size of the payload which includes both the encrypted data and the optional additional MAC text */
-    uint8_t   reserved[12];                   /*  4: Reserved bits */
+    uint32_t  payload_size;                   /*  0: Size of the payload which includes the encrypted data: payload[] */
+    uint8_t   reserved[SGX_SEAL_IV_SIZE];     /*  4: Reserved bits used as iv */
```

Comment:
Although the value of the used IV in tseal library is the same as this reserved bits array, but they are indeed different buffers.  This reserved array is actually to make the structure 16byte aligned.

Reply:
See the sample code, it's used as IV:
https://github.com/intel/linux-sgx/blob/master/SampleCode/LocalAttestation/EnclaveInitiator/EnclaveMessageExchange.cpp#L199
https://github.com/intel/linux-sgx/blob/master/SampleCode/LocalAttestation/EnclaveInitiator/EnclaveMessageExchange.cpp#L207

Reply:
OK. Thanks for the explanation. This usage may not be aligned with the initial design of the structure. But the functionality is correct.